presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java

```diff
@@ -68,82 +70,94 @@ private HiveQueryRunner()
     private static final String TPCH_BUCKETED_SCHEMA = "tpch_bucketed";
     private static final DateTimeZone TIME_ZONE = DateTimeZone.forID("America/Bahia_Banderas");
 
-    public static DistributedQueryRunner createQueryRunner(TpchTable<?>... tables)
+    public static DistributedQueryRunner create()
             throws Exception
     {
-        return createQueryRunner(ImmutableList.copyOf(tables));
+        return builder().build();
     }
 
-    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables)
-            throws Exception
+    public static Builder builder()
     {
-        return createQueryRunner(tables, ImmutableMap.of(), Optional.empty());
+        return new Builder();
     }
 
-    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> extraProperties, Optional<Path> baseDataDir)
-            throws Exception
+    public static class Builder
+            extends DistributedQueryRunner.Builder
```

Comment:
since you inherit parent builder, you should pass your own `Builder` type there and the parent builder should return `T` from build methods... otherwise code using builder needs to use methods in certain specific order, not the way builder should work

follow-up maybe

Reply:
I was thinking about this. It looks like a follow-up

Reply:
#3015